### PR TITLE
Reference management models template

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to #3145

### CODE REVIEW

File: Reference/Management/Models/Template-v9.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: Template Model
```
Comment (busrasengul) [inline Reference/Management/Models/Template-v9.md:4]:
```suggestion
meta.Title: "Template Model"
```

File: Reference/Management/Models/Template-v9.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+meta.Description: Represents a Template file.
```
Comment (busrasengul) [inline Reference/Management/Models/Template-v9.md:5]:
```suggestion
meta.Description: "Represents a Template file."
```

**Review by busrasengul — CHANGES_REQUESTED** [review]
I've requested meta changes again then this is good to merge

File: Reference/Management/Models/Template-v9.md (hunk 2 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Template-v9.md]: @busrasengul Updated with you suggestions

**Pull request merged**